packages/grid/x-data-grid-premium/src/tests/exportExcel.DataGridPremium.test.tsx

```diff
@@ -16,7 +16,7 @@ import { spyApi } from 'test/utils/helperFn';
 
 const isJSDOM = /jsdom/.test(window.navigator.userAgent);
 
-describe('<DataGridPremium /> - Export Excel', () => {
+describe('<DataGridPremium /> - Export excel', () => {
```

Comment:
Shouldn't it remain capitalized, because it is a product name? 🤔 
```suggestion
describe('<DataGridPremium /> - Export Excel', () => {
```